x-pack/plugins/secops/public/lib/lib.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IModule, IScope } from 'angular';
+import { NormalizedCacheObject } from 'apollo-cache-inmemory';
+import ApolloClient from 'apollo-client';
+import React from 'react';
+import { Observable } from 'rxjs';
+
+export interface AppFrontendLibs {
+  framework: AppFrameworkAdapter;
+  apolloClient: AppApolloClient;
+  observableApi: AppObservableApi;
+}
+
+export type AppTimezoneProvider = () => string;
+
+export type AppApolloClient = ApolloClient<NormalizedCacheObject>;
+
+export interface AppFrameworkAdapter {
+  // Insstance vars
+  appState?: object;
+  dateFormat?: string;
+  kbnVersion?: string;
+  scaledDateFormat?: string;
+  timezone?: string;
+
+  // Methods
+  setUISettings(key: string, value: any): void;
+  render(component: React.ReactElement<any>): void;
+  renderBreadcrumbs(component: React.ReactElement<any>): void;
+}
+
+export interface AppFramworkAdapterConstructable {
```

Comment:
nit: Misspelling. This should be `AppFrameworkAdapterConstructibe` It is missing an `e` and `Constructible` is missing an i